### DISCUSSION TITLE
Unable to get Syft to run

### DISCUSSION THREAD

**Discussion by raaghavvd** [OpenMined/courses#351] [Course: Foundations of Private Computation]
Hey, I'm getting the following error when i try to install this :
pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft

Collecting syft
  Cloning https://github.com/OpenMined/PySyft (to revision dev) to /private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft
    ERROR: Command errored out with exit status 1:
     command: /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/setup.py'"'"'; __file__='"'"'/private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-pip-egg-info-2ev8c0i9
         cwd: /private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tokenize.py", line 392, in open
        buffer = _builtin_open(filename, 'rb')
    FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/setup.py'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.Collecting syft
  Cloning https://github.com/OpenMined/PySyft (to revision dev) to /private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft
    ERROR: Command errored out with exit status 1:
     command: /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/setup.py'"'"'; __file__='"'"'/private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-pip-egg-info-2ev8c0i9
         cwd: /private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tokenize.py", line 392, in open
        buffer = _builtin_open(filename, 'rb')
    FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/2n/fcpjb5192pggds1xn7bv9qbm0000gn/T/pip-install-uld2cn6y/syft/setup.py'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

Can some one help?

**Comment by znreza** [comment]
Hi @raaghavvd based on your question [here](https://github.com/OpenMined/courses/discussions/354), I am assuming you already solved this issue and got passed this error?

**Reply by znreza** [reply]
If it's still not solved, could you please try [this](https://github.com/OpenMined/courses/discussions/307#discussioncomment-839323) way that is suggested here? You can also directly install [syft 0.5](https://github.com/OpenMined/PySyft/tree/syft_0.5.0) as well.
